CONTRIBUTORS.md

```diff
@@ -2,9 +2,10 @@ By adding your name and email address you agree to the terms of Lyft's CLA.
 Please use the following format: Name (email@address.com)
 
 1. Ryan Lane (ryan@ryandlane.com)
-1. Evelio Tarazona (evelio@evelio.info)
-1. Bart Whiteley (bwhiteley@gmail.com)
-1. skyfishjy (skyfish.jy@gmail.com)
-1. Aleksei Zakharov (alexey.v.zaharov@gmail.com)
-1. Ostap Taran (austinate@me.com)
-1. Zachary Perrault (zach.perrault@gmail.com)
+2. Evelio Tarazona (evelio@evelio.info)
+3. Bart Whiteley (bwhiteley@gmail.com)
+4. skyfishjy (skyfish.jy@gmail.com)
+5. Aleksei Zakharov (alexey.v.zaharov@gmail.com)
+6. Ostap Taran (austinate@me.com)
+7. Zachary Perrault (zach.perrault@gmail.com)
+8. Prayag Verma (prayag.verma@gmail.com)
```

Comment:
You can make all of these 1. Markdown renders them correctly regardless of the number